rust-keybroker/certs/veraison-root.crt

```diff
@@ -0,0 +1,10 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is this the root cert for the Docker deployment of Veraison? If so, then it won't be suitable for the Linaro service. This means that we have a clash of defaults, because the default API endpoint is the Linaro one.

Reply:
This is the certificate used in the example from rust-apiclient.

I aggree we should have sane and sound defaults. If the default verifier is Linaro's, then we should either ship the relevant certificate or keybroker-server should get it from LetEncrypt or whoever else.